beancount_reds_importers/libreader/csvreader.py

```diff
@@ -108,10 +108,17 @@ def convert_columns(self, rdr):
         # fixup currencies
         def remove_non_numeric(x):
             return re.sub("[^0-9\.-]", "", str(x).strip())  # noqa: W605
+
+        def add_decimal(x):
+            if '.' not in x:
+                return x+".00"
```

Comment:
For international compatibility, it would be best to use `locale` to print this, since decimal separator characters vary based on one's region. You may already be familiar with it, but if not, a bit of digging [into this](https://stackoverflow.com/questions/31910741/how-can-i-locale-format-a-python-decimal-and-preserve-its-precision) might provide a good solution. In particular, the setting of `locale` needs to be figured out. I'd imagine Beancount does this already, so looking at what it does might provide the right solution here. Thoughts?